src/main/java/objectsBasicsMandatoryPracticalTasks/Dogs/Size.java

```diff
@@ -0,0 +1,25 @@
+package objectsBasicsMandatoryPracticalTasks.Dogs;
+
+import java.util.Scanner;
+
+class Size {
+    private String[] arrSize;
+
+    String[] inputSize(boolean dummy) {
```

Comment:
Class `Size` should not be responsible to get values from console, so this method should be a part of `DogsDemo`